src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-template-spec/main.combined.bicep

```diff
@@ -0,0 +1,8 @@
+resource templatespec 'Microsoft.Resources/templateSpecs@2019-06-01-preview' = {

```

Comment:
Can you please camel case the identifier? e.g. templateSpec

Reply:
Good catch, resolved!